### DISCUSSION TITLE
How can i start it while using ssh?

### DISCUSSION THREAD

**Discussion by velainur** [rejetto/hfs#503] [Q&A] [answered]
I have hfs linux version in a Ubuntu server.

I can access it via ssh, but when i try to run it, i cannot do it even if i export DISPLAY.

How can i run remotely login via ssh to the device which holds hfs?
(and quit from ssh session leaving the hfs working)

Regards,

**Comment by rejetto** [comment]
what you mean by "cannot do it" ? describe the problem.
is it that when you exit ssh the program is closed?

**Comment by velainur** [comment]
Ok, sorry.... let me explain the process.

1.- Login inside the device with ssh client.
2.- I can export or not DISPLAY, no difference in the result.
3.- i run the hfs with "./hfs &" command.
At this moment, i can navigate the hfs with no problem.
4.- now i push enter 3-4 times... and i get the process stopped.
"[1]+  Stopped                 ./hfs"
Its a weird thing, since if i try to run it again, the console says:
"http port 8080 busy: hfs"
 ">> try specifying a different port, enter this command: config port 1080"
5.- Now if i press enter some times:
"[2]+  Stopped                 ./hfs"

If i try to check the process, it still working (appears in the ps list) but no one is listening the service port)

Anyway, i lost the connectivity to the server.

Regards,

David

**Comment by rejetto** [comment]
i don't know this stuff, but it doesn't seem hfs related.
i just googled, and saw that you probably got your process "suspended", and you can resume with "fg" command.
i don't know what's the best way to handle your case, i know a friend who uses "screen" command to leave programs running in a terminal.

**Comment by velainur** [comment, marked as answer]
I have tested the screen comand, but finnally, i think its easier with:
(i didnt like so much "screen" command due complexity of how to control process/screen)

"nohup ./hfs &"

Everything now works even if i close the ssh session.

Thank you so much for the help!!

**Reply by rejetto** [reply]
thanks for sharing it
